docs/full-node/devnet.md

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 50
+title: Devnet
+sidebar_label: Devnet
+---
+
+import UntranslatedPageText from "@site/src/components/UntranslatedPageText";
+
+<UntranslatedPageText />
+
+# Create a local Devnet
+
+## Using Docker
+
+```sh
+npx @alephium/cli@latest devnet start
```

Comment:
Should this be under the "using docker" headline?

Reply:
oh it indeed doesn't launch docker, we could have an "Automatic way` section?

Reply:
I mean, we could rename:
*  `Using Docker` -> `Automatically with our development tool`
* `Using jar files` ->  `Manually using jar files`